hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/FileSystem.java

```diff
@@ -636,18 +636,19 @@ public static void closeAll() throws IOException {
   public static void closeAllForUGI(UserGroupInformation ugi)
   throws IOException {
     if (LOGGER.isDebugEnabled()) {
-      debugLogFileSystemClose("closeAllForUGI", "UGI: " + ugi.toString());
+      debugLogFileSystemClose("closeAllForUGI", "UGI: " + ugi);
     }
     CACHE.closeAll(ugi);
   }
 
   private static void debugLogFileSystemClose(String methodName, String additionalInfo) {
     StackTraceElement callingMethod = new Throwable().fillInStackTrace().getStackTrace()[2];
-    LOGGER.debug(
-        "FileSystem." + methodName + "() called by method: "
-            + callingMethod.getClassName() + "." + callingMethod.getMethodName()
-            + "(" + callingMethod.getFileName() + ":" + callingMethod.getLineNumber() + "); "
-            + (additionalInfo != null ? additionalInfo : ""));
+    LOGGER.debug("FileSystem.{}() called by method: {}.{}({}:{}); {}", methodName, callingMethod.getClassName(),
```

Comment:
* include URI of filesystem
* pass in callingMethod and let it's toString() include all the information; no need to replicate -especially as toString handles file/line unknown.

Reply:
I included the URI in the case of close().